README.md

```diff
@@ -29,30 +29,57 @@ repos:
             "URL",
             "--env-file-path",
             "ENV_FILE_PATH",
-            "--enabled-entities",
-            "ENTITY-1",
-            "ENTITY-2",
           ]
         verbose: true
 
 ```
 4. Run 'pre-commit install' from inside the git repo where you want to use this hook.
 5. Replace 'URL' with the url of where your container is hosted.\
-     eg. http://localhost:8080/v3/process_text for a container running locally or https://api.private-ai.com/deid/v3/process_text for Private AI's cloud endpoint.
+     eg. http://localhost:8080/v3/process_text for a container running locally.
 6. Create a .env file and add your API_KEY like so:\
     API_KEY=`<put your API KEY here>`
 7. Replace 'ENV_FILE_PATH' with the path to your .env file.
-8. If you wish to add the optional argument --enabled-entities replace 'ENTITY-1', 'ENTITY-2' with the entities you wish to mark as PII
+
+That's it! You're all set for safe commits!
+
+## Additional Configuration (Optional)
+1. If you want to treat only certain entities as PII, while ignoring others you can add the optional argument ```--enabled-entities``` followed by the entities you wish to mark as PII.
     (Recommended to add this argument currently)\
-    You may add any entities from the following list: https://docs.private-ai.com/entities/ 
-   If you choose not to provide this option, all entities will be enabled by default. 
+2. If you want to detect all entities except a few, you can provide the optional argument ```--blocked-list``` followed by the entities you do NOT wish to be marked as PII.
+
+For both these configurations, you may add any entities from the following list: https://docs.private-ai.com/entities/ 
```

Comment:
Sorry. I just noticed that this is actually wrong. The code is using the deid `block` list feature which is not what it is being described.